### ISSUE TITLE
Domestic rubbish including bath outside ground floor flat 29-39 Blackthorn Road block

### ISSUE THREAD

**Issue by Thurgarton66** [theparksilford/Issues#60]
### Property Details

29-39 Blackthorn Road

### Contact Details

Thurgarton

### Type of problem?

Bulk Rubbish/Dumping

### Since when this issue started?

16/10/2023

### Issue Description?

There is a pile of domestic rubbish including a bath outside the bottom right hand side ground floor flat in the block 29-39 Blackthorn Road. 

I asked the resident when it was going to be removed and they said unconvincingly  today but didn’t know where. 

The danger is it will end up in the communal store. 

It looks like a change of tenancy. Please contact the owner to arrange responsible removal or otherwise charge them for removal. 
![IMG_3420](https://github.com/theparksilford/Issues/assets/105227854/e80cd11f-1854-4ec0-bb8f-45e4d0755bb6)
![IMG_3421](https://github.com/theparksilford/Issues/assets/105227854/725067cd-d9a0-4115-afc5-022cefda2840)

### Logged By

Leaseholder or FreeHolder

**Comment by jeremydoran** [comment]
Hello,
Thank you for letting us know.
Are you able to confirm the exact flat where you believe this has come from and the flat of the resident you spoke to so this can be followed up?
Thanks

**Comment by jeremydoran** [comment]
Yes we do, I will look into this. It can be easier if we know which flat.
We will look into this and contact the owner. 
Thanks.

**Comment by jeremydoran** [comment]
This rubbish has now been removed.
